src/components/Display.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Display = props => {
+  const { result } = props;
+  Display.propTypes = {
+    result: PropTypes.string,
+  };
+
+  Display.defaultProps = {
+    result: '0',
+  };
```

Comment:
This should be defined outside the display function because ideally, you are calling the function variable